chat-plugins/trivia.js

```diff
@@ -164,7 +164,7 @@ var Trivia = (function () {
 	}
 
 	Trivia.prototype.addParticipant = function (output, user) {
-		if (this.phase !== 'signup') return output.sendReply("There is no trivia game in its signup phase.");
+		if ((user.locked || this.room.isMuted(user)) && !user.can('bypassall')) return this.sendReply("You cannot do this while unable to talk.");
```

Comment:
Hmmm, I think we had a function called `this.canTalk` that does all the checks of whether a user can talk in a room. You might want to look into that.